components/ui-api-layer/internal/domain/servicecatalog/clusterserviceplan_converter.go

```diff
@@ -91,12 +97,28 @@ func (p *clusterServicePlanConverter) unpackCreateParameterSchema(raw []byte) (g
 }
 
 func (p *clusterServicePlanConverter) extractCreateSchema(raw []byte) (map[string]interface{}, error) {
+	raw = bytes.Trim(raw, "\x00")
 	extracted := make(map[string]interface{})
 
 	err := json.Unmarshal(raw, &extracted)
 	if err != nil {
 		return nil, errors.Wrap(err, "while extracting creation parameter schema")
 	}
+	if p.isEmptyCreateParameterSchema(extracted) {
+		return nil, nil
+	}
 
 	return extracted, nil
 }
+
+func (p *clusterServicePlanConverter) isEmptyCreateParameterSchema(schema map[string]interface{}) bool {
+	properties := schema["properties"]
+	if properties != nil && len(properties.(map[string]interface{})) != 0 {
```

Comment:
`properties.(map[string]interface{})`

Always check if the conversion is OK

Reply:
I fix it 👌